pkg/driver/driver_openstack.go

```diff
@@ -453,3 +453,17 @@ func (d *OpenStackDriver) recentImageIDFromName(client *gophercloud.ServiceClien
 	}
 	return "", fmt.Errorf("could not find an image id for image name %s", imageName)
 }
+
+// GetVolNames parses volume names from pv specs
+func (d *OpenStackDriver) GetVolNames(specs []corev1.PersistentVolumeSpec) ([]string, error) {
+	names := []string{}
+	for _, spec := range specs {
```

Comment:
Same shallow copy issue as [above](https://github.com/gardener/machine-controller-manager/pull/275/files#diff-face99aebd3cd79577003f7df747c863R335) but with less impact because the number of volumes per pod will be less.